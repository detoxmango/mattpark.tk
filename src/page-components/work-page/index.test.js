import React from 'react'
// import renderer from 'react-test-renderer'
import WorkPage from './index'

describe('WorkPage', () => {
    // let tree

    beforeEach(() => {
        // tree = renderer.create(<WorkPage />).toJSON()
    })
    it('passes', () => {
        expect(true).toBe(true)
    })
    // it("renders correctly", () => {
    //   expect(tree).toMatchSnapshot()
    // })
})
