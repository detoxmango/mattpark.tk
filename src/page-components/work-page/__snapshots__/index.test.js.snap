// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkPage renders correctly 1`] = `
<Unknown
  classes={
    Object {
      "placeholder": "Component-placeholder-2",
      "root": "Component-root-1",
    }
  }
/>
`;
